app/models/dataset.rb

```diff
@@ -168,6 +169,10 @@ def timeseries?
     %w[annually quarterly monthly].include?(frequency)
   end
 
+  def public_updated_at
+    most_recently_updated_datafile_timestamp || self.updated_at
```

Comment:
I might be missing some context, should this be the most recent of the two dates? Or is public_updated_at solely about the update time of the data files, and the metadata record updated_at is just a backup?

Reply:
Okay. I was just concerned about legacy import, but that should be okay as the updated_at should mostly be the same.  Thanks for the clarification.